common/build/eslint-config-fluid/base.js

```diff
@@ -135,15 +135,11 @@ module.exports = {
 			},
 		],
 
-		// eslint-plugin-import
+		// #region eslint-plugin-import
+
 		"import/no-default-export": "error",
 		"import/no-deprecated": "off",
-		"import/no-extraneous-dependencies": [
-			"error",
-			{
-				devDependencies: ["**/*.spec.ts", "src/test/**"],
-			},
-		],
+		"import/no-extraneous-dependencies": "error",
```

Comment:
The allowance for test code to use dev dependencies was moved into `minimal-deprecated`'s test code overrides block. Simplifies things a bit. Note `base.js` is not directly available to consumers of this package, it just serves as a shared base for our actual configs. So, this is not a breaking change.